src/Pages/CreatorPage.js

```diff
@@ -1,7 +1,16 @@
 import React, { useEffect, useState } from 'react';
 import { useParams } from 'react-router-dom';
-import AuthorInfo from '../Components/CreatorPage/Creator';
-import CreatedWorksCard from '../Components/CreatorPage/OtherWorks';
+import AuthorInfoCard from '../Components/creatorPage/AuthorInfoCard';
```

Comment:
Ahhhh I think maybe the naming conventions weren't fixed at the point you branched off, but every place where an 'AuthorCard' or a AuthorId is referenced, author needs to be replaced with 'creator'! These components will be used for both authors and illustrators so creators is more general and less confusing for other people who view the code down the line. The file names and variables will need to be changed appropriately. Try pulling from main, but otherwise you may have to do this by hand, I'm so sorry for making you repeat other work TT